packages/collector/src/tracing/instrumentation/process/selfPath.js

```diff
@@ -5,20 +5,7 @@
 
 'use strict';
 
-const { uninstrumentedFs: fs } = require('@instana/core');
 const path = require('path');
 
-/** @type {import('@instana/core/src/core').GenericLogger} */
-let logger;
-logger = require('../../../logger').getLogger('tracing/selfPath', newLogger => {
-  logger = newLogger;
-});
-
 // @ts-ignore - Cannot redeclare exported variable
 exports.immediate = path.join(__dirname, '..', '..', '..', 'immediate.js');
-
-if (!fs.existsSync(exports.immediate)) {
```

Comment:
We print the warning on the caller component.

https://github.com/instana/nodejs/blob/v4.6.1/packages/collector/src/tracing/instrumentation/process/edgemicro.js#L36

https://github.com/instana/nodejs/blob/v4.6.1/packages/collector/src/tracing/instrumentation/process/childProcess.js#L49